### DISCUSSION TITLE
ScrollTo is buggy for me

### DISCUSSION THREAD

**Discussion by VPValter** [shipshapecode/shepherd#2165] [Q&A]
Hello everyone

I'm using `scrollTo: { behavior: 'smooth', block: 'center' }`, but it's often buggy and doesn't scroll the element to the center, it just scrolls to where the popup is barely visible on the edge, with the element it points to often being outside the view if it's above or below (or behind a fixed header or something like that).
Does anyone else have problems with scrollTo ?
(the version is 8.3.1)

**Comment by RobbieTheWagner** [comment]
@VPValter that is quite an old version of Shepherd. I am not even sure if the API was the same then.
